### PR TITLE
Feature/main leaderboard

### PR DESCRIPTION
**Pull request by ArnaudPoletto**

### CODE REVIEW

File: app/src/main/java/sdp/moneyrun/menu/MainLeaderboardListAdapter.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        if(position < rank.length){
+            text_position = rank[position];
+        }else{
+            text_position = " " + (position + 1);
+        }
+
+        player_position.setText(text_position);
+        player_name.setText(String.valueOf(player.getName()));
+        if(user.equals(player)){
+            player_name.setTextColor(Color.rgb(255, 204, 51));
```
Comment (gauthierboeshertz) [inline app/src/main/java/sdp/moneyrun/menu/MainLeaderboardListAdapter.java:51]: Magic values should be changed to private final ..

**Review by gauthierboeshertz — CHANGES_REQUESTED** [review]

File: app/src/main/java/sdp/moneyrun/menu/MainLeaderboardListAdapter.java (hunk 1 above)
Comment (ArnaudPoletto) [reply app/src/main/java/sdp/moneyrun/menu/MainLeaderboardListAdapter.java]: done

File: app/src/main/java/sdp/moneyrun/menu/MainLeaderboardListAdapter.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
+import sdp.moneyrun.R;
+import sdp.moneyrun.player.Player;
+
+public class MainLeaderboardListAdapter extends ArrayAdapter<Player> {
+
+    private final String[] rank = {"\uD83E\uDD47", "\uD83E\uDD48", "\uD83E\uDD49"};
```
Comment (Tesa27) [inline app/src/main/java/sdp/moneyrun/menu/MainLeaderboardListAdapter.java:20]: these values are not really straight forward a comment about what each value represents would be useful

**Review by gauthierboeshertz — APPROVED** [review]

File: app/src/main/java/sdp/moneyrun/menu/MainLeaderboardListAdapter.java (hunk 2 above)
Comment (ArnaudPoletto) [reply app/src/main/java/sdp/moneyrun/menu/MainLeaderboardListAdapter.java:21]: done

**Review by Tesa27 — APPROVED** [review]

**Pull request merged**